plasma_framework/test/helpers/testlang.js

```diff
@@ -0,0 +1,12 @@
+const TransactionOutput = require("./transaction.js").TransactionOutput
```

Comment:
Minor: I think we can do 
```
const { TransactionOutput } = require('./transaction.js');
```

like the style we do for `openzeppelin-test-helpers`.

Reply:
done